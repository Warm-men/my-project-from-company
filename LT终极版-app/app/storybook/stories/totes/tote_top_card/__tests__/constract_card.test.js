import React from 'react'
import ImageTouchCard from '../image_touch_card'
import renderer from 'react-test-renderer'

describe('constract card', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<ImageTouchCard type={'constract'} onClick={jest.fn()} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
