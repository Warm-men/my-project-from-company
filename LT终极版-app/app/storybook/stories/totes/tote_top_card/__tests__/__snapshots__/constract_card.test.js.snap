// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constract card renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFF",
      "borderColor": "#f3f3f3",
      "borderRadius": 4,
      "borderWidth": 1,
      "elevation": 1,
      "flex": 1,
      "marginHorizontal": 32,
      "marginTop": 16,
      "padding": 24,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.08,
      "shadowRadius": 4,
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  />
</View>
`;
